src/Motor.Extensions.Diagnostics.Telemetry/TelemetryHostBuilderExtensions.cs

```diff
@@ -115,54 +94,26 @@ public static IMotorHostBuilder ConfigureOpenTelemetry(this IMotorHostBuilder ho
     {
         Sampler sampler = options.SamplingProbability switch
         {
-            > 0.9999 => new AlwaysOnSampler(),
+            > 1 => new AlwaysOnSampler(),
```

Comment:
nit: should we check here `>= 1`  instead for the sake of clarity and performance, even if for `= 1` this is also handled in `TraceIdRatioBasedSampler`?